packages/platform/package.json

```diff
@@ -26,5 +26,7 @@
   "dependencies": {
     "nitropack": "^1.0.0",
     "@analogjs/vite-plugin-angular": "latest"
-  }
+  },
+  "generators": "./generators.json",
```

Comment:
suggestion: let's also add `schematics` property here pointing to the same `generators.json` with the above change